common/script/index.coffee

```diff
@@ -1740,14 +1744,22 @@ api.wrap = (user, main=true) ->
           dailyChecked += 1
         else
           # dailys repeat, so need to calculate how many they've missed according to their own schedule
+          console.log("task: " + task.text) # XXX Alys to remove these
           scheduleMisses = 0
+          keepGoing = true # XXX Alys to improve how we break out of _.times
           _.times daysMissed, (n) ->
-            thatDay = moment(now).subtract({days: n + 1})
-            if api.shouldDo(thatDay.toDate(), task, user.preferences)
-              scheduleMisses++
-              if user.stats.buffs.stealth
-                user.stats.buffs.stealth--
-                EvadeTask++
+            if keepGoing
+              console.log("n: " + n)
+              thatDay = moment(now).subtract({days: n + 1})
+              if api.shouldDo(thatDay.toDate(), task, user.preferences)
+                console.log("damage")
+                scheduleMisses++
+                if user.stats.buffs.stealth
+                  user.stats.buffs.stealth--
+                  EvadeTask++
+                if multiDaysCountAsOneDay
+                  console.log("end")
+                  keepGoing = false
```

Comment:
`keepGoing` is how I "break out" of the `_.times` loop but it sickens me. 
What is the preferred way to do that? Googling and testing gave me nothing (`break` fails, `return false` fails). From what I've read, I'm not sure it's possible. Should I replace `_.times` with an old-style `for` loop?